policies/rudderc/Cargo.toml

```diff
@@ -32,6 +32,7 @@ tempfile = "3"
 rudder_commons = { path = "../rudder-commons" }
 rudder_cli = { path = "../rudder-cli" }
 zip = { version = "2.1.3", default-features = false, features = ["deflate", "time"] }
+pretty_assertions = "1"
```

Comment:
Most likely not needed

Reply:
indeed, it is already a dev dep and should not be needed here